index.js

```diff
@@ -26,23 +31,23 @@ function getAwsPath(bucket) {
 
 function logError(error) {
     console.log('error in ghost-s3', error);
-};
+}
 
 function logInfo(info) {
     console.log('info in ghost-s3', info);
-};
+}
 
 function getTargetDir() {
```

Comment:
The `BaseStore` from `ghost` has an implementation for this, so it can be removed, along with the `moment` require. It's used in `S3Store.prototype.save` where the reference has to be changed to `this.getTargetDir()`.